react/components/FooterSocialNetworkList.js

```diff
@@ -14,7 +14,7 @@ const FooterSocialNetworkItem = ({ imageSrc, url }) => {
   }
 
   return (
-    <a href={url} target="_blank" className="c-muted-1">
+    <a href={url} target="_blank" className={`${footer.socialNetworkList} c-muted-1`}>
```

Comment:
I know this is not related to your PR, but since its low cost to solve, please add `rel="noopener"` to this link and take a look at [here](https://developers.google.com/web/tools/lighthouse/audits/noopener)

Reply:
Thanks for the tip. I didnt know about the relationship between `rel` and anchor sites. . 👨‍🏭